CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/AttributeEditorPanel.java

```diff
@@ -212,21 +207,13 @@ public AttributeEditorPanel(AttributeEditorTopComponent parent) {
         }
 
         scrollPane.setContent(titledPaneHeadingsContainer);
-        BoxLayout layout = new BoxLayout(this, BoxLayout.Y_AXIS);
-        setLayout(layout);
-        add(container);
         scrollPane.setFitToWidth(true);
 
-        final int height = this.getPreferredSize().height;
-
         Platform.setImplicitExit(false);
         Platform.runLater(() -> {
             root = new StackPane();
-
-            final Scene scene = new Scene(root, 0, height);
-            scene.getStylesheets().add(JavafxStyleManager.getMainStyleSheet());
-            scene.rootProperty().get().setStyle(String.format("-fx-font-size:%d;", FontUtilities.getOutputFontSize()));
-            scene.getStylesheets().add(AttributeEditorPanel.class.getResource(DARK_THEME).toExternalForm());
+            root.getStylesheets().add(JavafxStyleManager.getMainStyleSheet());
```

Comment:
This should be handled by the framework - it'll automatically apply the default stylesheet, and additionally apply a stylesheet specified by `JavaFxTopComponent.createStyle`. These styles will cascade down from the top component to its children (at least that's how it's supposed to work.

Reply:
It seems to be working fine after putting on the correct style sheet either way. Next commit will have the fixes in it 👍